### PR TITLE
add new step

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

**Review by MadayAlcala — APPROVED** [review]
Great!

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 1]
```diff
@@ -104,4 +104,11 @@ public void iGetASchema(String schemaName) {
     public void afterClass() {
         context.setId(null);
     }
+
+    @When("I send a request")
+    public void iSendARequest() {
```
Comment (LimbertVargas) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java:112]: please add documentation

Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java:112]: solved.

**Review by LimbertVargas — APPROVED** [review]
that's all... good job

**Pull request merged**